### PR TITLE
Issue #3323770: Add a ddev and gitpod enabled project template

### PR DESCRIPTION
**Pull request by StavoveiC**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
         "composer/installers": "^1.9",
         "drupal/core-composer-scaffold": "^9",
         "drupal/core-project-message": "^9",
-        "drupal/core-recommended": "^9",
-        "drush/drush": "^10.3"
+        "drush/drush": "^11",
+        "psr/container": "^2.0.2"
```
Comment (fago) [inline composer.json:22]: can we remove this?

File: composer.json [hunk 2]
```diff
@@ -87,5 +87,9 @@
                 "      composer remove drupal/core-project-message"
             ]
         }
+    },
+    "require-dev": {
+        "drupal/core-recommended": "^10",
```
Comment (fago) [inline composer.json:92]: should be in main require, why dev?

File: composer.json [hunk 3]
```diff
@@ -87,5 +87,9 @@
...
+        "drupal/lupus_decoupled": "^1.0@alpha"
```
Comment (fago) [inline composer.json:93]: should be in main require, why dev?

File: composer.lock [hunk 4]
```diff
@@ -999,762 +891,855 @@
...
             "support": {
                 "issues": "https://github.com/dflydev/dflydev-dot-access-data/issues",
-                "source": "https://github.com/dflydev/dflydev-dot-access-data/tree/master"
+                "source": "https://github.com/dflydev/dflydev-dot-access-data/tree/v3.0.2"
             },
-            "time": "2017-01-20T21:14:22+00:00"
+            "time": "2022-10-27T11:44:00+00:00"
         },
         {
-            "name": "doctrine/annotations",
```
Comment (fago) [inline composer.lock:1007]: not sure, since this is a template, should we drop the lockfile so upon installation latest packages get picked?

**Review by fago — CHANGES_REQUESTED** [review]

File: .gitpod.yml [hunk 5]
```diff
@@ -6,7 +6,8 @@ tasks:
   - init: |
       ddev start -y
       ddev composer install
-      ddev drush si -y --account-pass=admin --site-name='ddev_gitpod' demo_umami
+      ddev drush si -y --account-pass=admin --site-name='ddev_gitpod' standard
+      ddev drush en lupus_decoupled_ce_api lupus_decoupled_cors lupus_decoupled_menu -y
```
Comment (mostepaniukvm) [inline .gitpod.yml:10]: tested in gitpod and these modules are enabled but not the main on  - lupus_decoupled

Can we change drush command and enable only  lupus_decoupled and let lupus_decoupled_* enable automatically

**Review by mostepaniukvm — CHANGES_REQUESTED** [review]

**Review by fago — APPROVED** [review]

**Pull request merged**